services/category.js

```diff
@@ -2,11 +2,15 @@ const Category = require('~/models/category')
 
 const categoryService = {
   getCategories: async (searchFilter, skip, limit) => {
-    return await Category.find(searchFilter).skip(skip).limit(limit).lean().exec()
+    const categories = await Category.find(searchFilter).skip(skip).limit(limit).lean().exec()
```

Comment:
i think it is redundant to declare variable here, just return await

Reply:
yeap, already changed